frontend/src/services/objectService.ts

```diff
@@ -108,5 +109,38 @@ export default {
     fd.append('file', object);
     return comsAxios().post(`${PATH}/${objectId}`, fd, config);
   },
+
+  
+  /**
+   * @function addPermissions
+   * Adds the given permissions to the object
+   * @param {string} objectId ID of the object to add permissions to
+   * @param {Array<Object>} data Array containing permissions to add
+   * @returns {Promise} An axios response
+   */
+  addPermissions(objectId: string, data: Array<Object>) {
+    return comsAxios().put(`${OBJECT_PERMISSION_PATH}/${objectId}`, data);
+  },
+
+  /**
+ * @function deletePermission
+ * Deletes the given permission from the object
+ * @param {string} objectId ID of the object to remove permissions from
+ * @param {Object} params Object containing the permission to remove
+ * @returns {Promise} An axios response
+ */
+  deletePermission(objectId: string, params: Object) {
+    return comsAxios().delete(`${OBJECT_PERMISSION_PATH}/${objectId}`, { params: params });
+  },
+
+  /**
+   * @function searchForPermissions
+   * Returns a list of permissions for the object
+   * @param {Object} params Optional object containing the data to filter against
+   * @returns {Promise} An axios response
+   */
+  searchForPermissions(objectId: string, params?: Object) {
+    return comsAxios().get(`${OBJECT_PERMISSION_PATH}/${objectId}`, { params: params });
+  },
```

Comment:
We have a permissionService.ts file - this logic should reside there instead of this file.

Reply:
This one actually unused, removing